lib/shoulda/matchers/active_model/numericality_matchers/even_number_matcher.rb

```diff
@@ -20,6 +20,11 @@ def allowed_type
           def diff_to_compare
             2
           end
+
+          def strict
```

Comment:
The other matchers inherit from NumericTypeMatcher which already inherits from ValidationMatcher.